hummingbot/strategy/avellaneda_market_making/avellaneda_market_making.pyx

```diff
@@ -355,7 +355,7 @@ cdef class AvellanedaMarketMakingStrategy(StrategyBase):
             lines.extend(["", "  No active maker orders."])
 
         volatility_pct = self._avg_vol.current_value / float(self.get_price()) * 100.0
-        if all((self._gamma, self._kappa, volatility_pct)):
+        if all((self._gamma, self._kappa, volatility_pct is not NaN)):
```

Comment:
I don't think this is correct. I do know that checking if a variable is `NaN`, we can't just use `a is not NaN` or `a != NaN`. We should use `math.isnan()` or its equivalent.

![image](https://user-images.githubusercontent.com/6755922/114972643-74265000-9eb1-11eb-9e99-5e8d2f18ba55.png)

Reply:
Change applied